### ISSUE TITLE
Request: Add option for private/blocked numbers

### ISSUE THREAD

**Issue by norrinr** [x13a/Silence#25]
Private numbers, or ones with blocked caller ID (in USA dialing with *67 prefix blocks your number to the recipient), still get through and ring. Can you add an option to block private numbers as well? My dialer _does_ have the option to block those calls on my Samsung phone, but I'm not sure if all dialers have that. It would be nice to have all blocking done in one place, anyway.

**Comment by x13a** [comment]
It can not be done..

```text
Calls with a Call.Details#getHandlePresentation() of TelecomManager#PRESENTATION_RESTRICTED, 
TelecomManager#PRESENTATION_UNKNOWN or TelecomManager#PRESENTATION_PAYPHONE 
presentation are not provided to the CallScreeningService.
```

You have to find something like this in the settings of your default `Phone` app.

<img src="https://user-images.githubusercontent.com/53379023/143320762-a4a4c364-5dd1-4368-aa4e-f21966569276.png" width="30%" height="30%">

**Comment by norrinr** [comment]
Got it. Thank you!

**Comment by x13a** [comment]
Hi,

TLDR: I was wrong, sorry. Now private numbers are blocked)

**Comment by norrinr** [comment]
Thanks for the note! Love this app.
